bookings/models.py

```diff
@@ -110,3 +142,24 @@ def retrieve(self, passed_parameters=None):
             self.source_id, passed_parameters=passed_parameters
         )
         return response_data.get("tickets", [])
+
+
+# TODO `PriceModel` has currency as well, but in tickets specs currency isn't mentioned
```

Comment:
We can use fixed `EUR` here for now. No other currencies are supported anyways.

Reply:
Hm well, there is no default in PriceModel, I think it's still safe to get currency from fare, that will be ofc `EUR` if I am not wrong, https://github.com/City-of-Helsinki/maritime-maas/pull/86/files#diff-8e5d59c82b371b9c786a0a402c48e2f257356142d26f7f589ffebb7d8c1c1cbdR48